Assets/Scripts/Charater Scripts/ShootingEnums.cs

```diff
@@ -0,0 +1,13 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class ShootingEnums : MonoBehaviour
```

Comment:
This should be separated. In this file there should only be enums, no classes. In this case we do not want to make enums an attribute in a class.